bazel/repositories.bzl

```diff
@@ -80,10 +80,21 @@ def python_deps(skip_targets):
             actual = "@jinja2_git//:jinja2",
         )
 
+def envoy_api_deps(skip_targets):
+  if 'envoy_api' not in skip_targets:
+    native.git_repository(
+        name = "envoy_api_git",
+        remote = "https://github.com/lyft/envoy-api.git",
+        commit = "959278cc35a89a4d2f1895f66a59c6b3de98d5e1",
+    )
+    native.bind(
```

Comment:
should we bind all the different libraries in this PR? Or do you want to bring them one by one as we start migrating to v2? 

Should we start some documentation on this for this PR?